packages/calcite-components/src/components/combobox/combobox.e2e.ts

```diff
@@ -889,8 +889,8 @@ describe("calcite-combobox", () => {
       await skipAnimations(page);
       await page.setContent(html`
         <calcite-combobox allow-custom-values selection-mode="single">
-          <calcite-combobox-item selected id="one" value="one" text-label="one"></calcite-combobox-item>
```

Comment:
Does the move of the `selected` attribute affect the test? If so, there might be an issue. If not, let's restore it. Applies to similarly updated tests.

Reply:
I restored the `selected` props.